kv/txn_coord_sender.go

```diff
@@ -322,8 +322,7 @@ func (tc *TxnCoordSender) Send(ctx context.Context, ba proto.BatchRequest) (*pro
 	// TxnID (if applicable). Begin a Trace which follows this request.
 	trace := tc.tracer.NewTrace(&ba)
 	defer trace.Finalize()
-	// TODO(tschottdorf): always "Batch"
-	defer trace.Epoch(fmt.Sprintf("sending %s", ba.Method()))()
+	defer trace.Epoch(fmt.Sprintf("sending %s", proto.Batch))()
```

Comment:
I don't really like this change from `.Method()` to a hard-coded `proto.Batch` (here and elsewhere, especially where it's combined with a literal like in dist_sender_test.go). I'd prefer either keeping a `.Method()` method on `BatchRequest` (similar to the one we have on `sql/driver.Request`, even though the only method there is `Execute`), or dispense with the constant altogether and just make it a string literal: `trace.Epoch("sending Batch")`.